### PR TITLE
Create Statefull Component

### PR DESCRIPTION
**Pull request by samizad**

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -1,32 +1,22 @@
...
+const Header = ({ title, onAdd, showAdd }) => {
+  return (
+    <div>
+      <header className="header">
+        <h1>{title}</h1>
+      </header>
+    </div>
+  );
+};
+///style  style={headingStyle} for styling// css in js
```
Comment (azedo) [inline src/components/Header.js]: Remove comments

File: src/components/Task.js [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
-            <p>{task.day}</p>
-            
-        </div>
-    )
-}
-
-export default Task
+import {FaTimes} from 'react-icons/fa'
+
+const Task = ({task, onDelete, onToggle}) => {
```
Comment (azedo) [inline src/components/Task.js]: Check formatting with prettier

File: src/components/Tasks.js [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
-                onToggle={onToggle}
-                />
-            ))}
-        </>
-    )
-}
-
-export default Tasks
+
+import Task from './Task'
```
Comment (azedo) [inline src/components/Tasks.js]: Check formatting

File: src/components/AddTask.js [hunk 4]
```diff
@@ -1,60 +1,60 @@
...
-<input type='submit'
- value='Save Task' 
- className='btn btn-block'
-  />
-</form>
-  ) 
-};
-
-export default AddTask;
+import React,{useState} from 'react';
```
Comment (azedo) [inline src/components/AddTask.js]: check formatting

File: src/components/TasksContainer.js [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+  }
+];
+
+const TasksContainer = (props) => {
+  const [tasks, setTasks] = useState(initialTasks);
+  const [showAddTask, setShowAddTask] = useState(false);
+  const onShow = () => {
+    setShowAddTask(!showAddTask);
+  };
+  //add tasks
```
Comment (azedo) [inline src/components/TasksContainer.js]: Here's a suggestion on how to improve this comment

```suggestion
  // This method creates a new task for the user

```

File: src/components/TasksContainer.js [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+  const addTask = (task) => {
+    //you should get it from backend
+    const id = Math.floor(Math.random() * 10000) + 1;
+    const newTask = { id, ...task };
+    console.log(newTask);
```
Comment (azedo) [inline src/components/TasksContainer.js]: Remove console log

File: src/components/TasksContainer.js [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+    setTasks([...tasks, newTask]);
+    console.log(tasks);
```
Comment (azedo) [inline src/components/TasksContainer.js]: Remove console log

File: src/components/TasksContainer.js [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+  };
+
+  //delete task
```
Comment (azedo) [inline src/components/TasksContainer.js]: If you keep this, try to improve the comment

File: src/components/TasksContainer.js [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+  const deleteHandler = (id) => {
+    setTasks(tasks.filter((task) => task.id !== id));
+  };
+  //toggle reminder
```
Comment (azedo) [inline src/components/TasksContainer.js]: Same here!

**Review by azedo — COMMENTED** [review]
Left some comments for you

File: src/components/Header.js (hunk 1 above)
Comment (samizad) [reply src/components/Header.js]: Done

File: src/components/Task.js (hunk 2 above)
Comment (samizad) [reply src/components/Task.js]: Done

File: src/components/Tasks.js (hunk 3 above)
Comment (samizad) [reply src/components/Tasks.js]: Done

File: src/components/TasksContainer.js (hunk 5 above)
Comment (samizad) [reply src/components/TasksContainer.js]: Done

**Pull request merged**